classes/Project/ProjectFactory.mjs

```diff
@@ -0,0 +1,86 @@
+import {Page} from "../Page/Page.mjs"
+import Project from "./Project.mjs"
+
+let err_out = Object.assign(new Error(), {
+  status: 500,
+  message: "Unknown Server error"
+})
+
+export default class ProjectFactory {
+  constructor(data) {
+    this.data = data
+  }
+
+  static async fetchManifest(url) { 
```

Comment:
Other libraries use the name like `loadManifest(url)` which may be preferable in case we start using caching more aggressively or allowing for other valid pipelines.